addons/tmf_loadouts/macros.inc

```diff
@@ -18,7 +18,8 @@ LIST_7("ACE_salineIV_500"), \
 LIST_5("ACE_salineIV_250"), \
 LIST_2("ACE_bodyBag"), \
 LIST_10("ACE_splint"), \
-LIST_5("ACE_salineIV")
+LIST_5("ACE_salineIV"), \
+"ACE_surgicalKit"
```

Comment:
Will this addition fit in current medic backpacks?

Reply:
> Will this addition fit in current medic backpacks?

It should be added to all backpacks, though it might in some cases replace something added later *after* the medikit, though that tends to be less important things like smoke grenades or body bags.

Reply:
If it's not replacing the medical items above then that is fine.